### DISCUSSION TITLE
Welcome to GEE-Courses Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/GEE-Courses#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by moshiour** [comment]
Hi,
I am Moshiour working as a data scientiest in energy company. I have been following your GEEmap and GeoSAM repo and would like to attend courses(paid) if you are arranging. Please let me know. Thank you in advance!/Moshiour

**Reply by giswqs** [reply]
All my courses are free. You can find them on YouTube. See https://youtu.be/ZvuX56mKHAI?si=bliz1ZgEP4zUtdTB

**Comment by moshiour** [comment]
Hi,
I was running the notebook: 11 export image.ipynb to export all (12) bands to a sngle Geotiff format from Copernicus/sentinel 2 data. I could not do it. 
Please show one example specific for sentinel 2 12 bands to single image Geotiff format.
Thank you in advance!
